src/CMakeLists.txt

```diff
@@ -311,7 +326,8 @@ target_link_libraries(
   Qt5::Widgets
   YAJL::YAJL
   ZIP::ZIP
-  ZLIB::ZLIB)
+  ZLIB::ZLIB
+  qtkeychain)
```

Comment:
This failed the build. Suggested change to fix it:
```suggestion
  qt5keychain)
```

Reply:
:facepalm: